src/Tabs/Tabs.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Flex from '../Flex';
+import Tab from './Tab';
+import { ENTER_KEY, SPACEBAR } from '../constants';
+
+class Tabs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const { activeTabLabel, children } = this.props;
+
+    this.state = {
+      activeTabLabel: activeTabLabel || children[0].props.label
```

Comment:
It feels like we're conflating the purpose of the `label` prop here for both, the user facing display of the tab and a unique identifier. Would it make sense to utilize an `id` instead on each tab?

Reply:
💯